### ISSUE TITLE
Profile mode for ios devices is broken when ios language is set to swift

### ISSUE THREAD

**Issue by christian-muertz** [flutter/flutter#24869]
The profile mode for ios devices is broken when the ios language is set to swift.

## Steps to Reproduce

<!-- Please tell us exactly how to reproduce the problem you are running into. -->

1. Create a new project using swift as the ios language:
 `flutter create -i swift profile_mode_broken`
2. Sign Provisioning Profile etc.
1. Start the app in profile mode on an ios device: 
`flutter run --profile`
## Logs

<!--
      Include the full logs of the commands you are running between the lines
      with the backticks below. If you are running any "flutter" commands,
      please include the output of running them with "--verbose"; for example,
      the output of running "flutter --verbose create foo".
-->

```
flutter run --profile
Launching lib/main.dart on Chris iPhone in profile mode...
Signing iOS app for device deployment using developer identity: "iPhone Developer: xxx@t-online.de (xxx)"
Starting Xcode build...                                          
 ├─Building Dart code...                              2,7s                                                                                                                                                                                                              
 ├─Generating dSYM file...                            0,3s                                                                                                                                                                                                              
 ├─Stripping debug symbols...                         0,0s                                                                                                                                                                                                              
 ├─Assembling Flutter resources...                    1,0s                                                                                                                                                                                                              
 └─Compiling, linking and signing...                  2,6s                                                                                                                                                                                                              
Xcode build done.                                            8,0s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    === BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Profile ===
    /Users/chris/Development/projects/client/ios/Runner/AppDelegate.swift:10:5: error: use of unresolved identifier 'GeneratedPluginRegistrant'
        GeneratedPluginRegistrant.register(with: self)
        ^~~~~~~~~~~~~~~~~~~~~~~~~

Could not build the precompiled application for the device.

Error running application on xxx iPhone. 
```

<!-- If possible, paste the output of running `flutter doctor -v` here. -->

```
flutter doctor -v
[✓] Flutter (Channel master, v0.11.13-pre.2, on Mac OS X 10.14.1 18B75, locale de-DE)
    • Flutter version 0.11.13-pre.2 at /Users/chris/Development/flutter
    • Framework revision 72926bdff7 (3 hours ago), 2018-11-29 09:32:11 -0800
    • Engine revision 72c7a75672
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /Users/chris/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.2

[✓] Android Studio (version 3.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 24.2.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[✓] IntelliJ IDEA Ultimate Edition (version 2018.2.4)
    • IntelliJ at /Users/chris/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 28.0.4
    • Dart plugin version 182.4323.44

[✓] Connected device (2 available)
    • Chris iPhone • 662d19f06a36af09a9bd0e64f7b0fdcd83fc5167 • ios • iOS 12.1
    • iPhone 7     • 16D3476F-4685-48C1-89FF-7D3D60F61C2A     • ios • iOS 12.1 (simulator)

• No issues found!

```

**Comment by adriancmurray** [comment]
I'm running into this same problem.

**Comment by adriancmurray** [comment]
```
Launching lib/main.dart on Adrian’s iPhone in profile mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: G739HAJ9R6
Starting Xcode build...
Xcode build done.                                           32.7s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/Adrian/dev/buskur/buskur/ios/Runner/AppDelegate.swift:10:5: error: use of unresolved identifier 'GeneratedPluginRegistrant'
        GeneratedPluginRegistrant.register(with: self)
        ^~~~~~~~~~~~~~~~~~~~~~~~~

Could not build the precompiled application for the device.

Error running application on Adrian’s iPhone.

```

```
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.13.6 17G65, locale en-US)
    • Flutter version 1.0.0 at /Users/Adrian/dev/flutter
    • Framework revision 5391447fae (10 days ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /Users/Adrian/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 31.1.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] VS Code (version 1.29.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.20.0

[✓] Connected device (1 available)
    • Adrian’s iPhone • 00008020-0002354A36F8002E • ios • iOS 12.1.1

• No issues found!

```

**Comment by panuavakul** [comment]
Same problem here too

**Comment by Donald-Chun** [comment]
Same problem here too

**Comment by zoechi** [comment]
@panuavakul @Donald-Chun "me too" comments are pointless https://github.com/flutter/flutter/wiki/Issue-hygiene#thumbs-up-reactions

**Comment by panuavakul** [comment]
@zoechi IMO it isn't pointless since it being attention to this Issue but I guess if there are rules against it then I won't do it. Thanks for letting me know.

**Comment by zoechi** [comment]
@panuavakul 

> being attention to this Issue

no, that's not the case. It only causes spammy messages to be sent to all involved or subscribed to the issue.
Upvotes on the initial issue bring attention to the issue.

**Comment by coreysprague** [comment]
For what it's worth, it is a simple fix. The build setting "Objective-C bridging header"  is blank for the profile configuration. You can copy the value from debug/release:

![image](https://user-images.githubusercontent.com/5420324/52492189-90b1f980-2b7d-11e9-8299-fb3cd6ac7c32.png)

**Comment by adriancmurray** [comment]
Works as advertised. Thanks, @coreysprague !

**Comment by Ogiwara-CostlierRain464** [comment]
Same issue, and solved with @coreysprague 's method.
Thanks!

**Comment by liyuqian** [comment]
@xster @dnfield : can we pull this fix inside our repo? I also encountered this today.

**Comment by dnfield** [comment]
If there's a PR open for it I'm happy to review.  Otherwise I can take a look when I'm back in the office on Tuesday.

**Comment by SeanOThomas** [comment]
Hmm, I'm actually still seeing this, except I'm just trying to run debug on a simulator. I confirmed that my bridging headers are set correctly. 

```
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:176:7: warning: 'UIGraphicsImageRendererFormat' is only available on iOS 10.0 or newer [-Wunguarded-availability]
          UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'UIKit' imported from /Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/Target Support Files/google_maps_flutter/google_maps_flutter-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIGraphicsImageRenderer.h:26:50: note: 'UIGraphicsImageRendererFormat' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 8.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(10.0)) @interface UIGraphicsImageRendererFormat : UIGraphicsRendererFormat
                                                     ^
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:176:7: note: enclose 'UIGraphicsImageRendererFormat' in an @available check to silence this warning
          UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:176:78: warning: 'defaultFormat' is only available on iOS 10.0 or newer [-Wunguarded-availability]
          UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
                                                                                 ^~~~~~~~~~~~~~
    In module 'UIKit' imported from /Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/Target Support Files/google_maps_flutter/google_maps_flutter-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIGraphicsRenderer.h:24:1: note: 'defaultFormat' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 8.0.0
    + (instancetype)defaultFormat API_DEPRECATED_WITH_REPLACEMENT("preferredFormat", tvos(10.0, 11.0)) API_AVAILABLE(ios(10.0), watchos(3.0));
    ^
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:176:78: note: enclose 'defaultFormat' in an @available check to silence this warning
          UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
                                                                                 ^~~~~~~~~~~~~~
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:176:48: warning: 'UIGraphicsImageRendererFormat' is only available on iOS 10.0 or newer [-Wunguarded-availability]
          UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'UIKit' imported from /Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/Target Support Files/google_maps_flutter/google_maps_flutter-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIGraphicsImageRenderer.h:26:50: note: 'UIGraphicsImageRendererFormat' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 8.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(10.0)) @interface UIGraphicsImageRendererFormat : UIGraphicsRendererFormat
                                                     ^
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:176:48: note: enclose 'UIGraphicsImageRendererFormat' in an @available check to silence this warning
          UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:178:7: warning: 'UIGraphicsImageRenderer' is only available on iOS 10.0 or newer [-Wunguarded-availability]
          UIGraphicsImageRenderer* renderer =
          ^~~~~~~~~~~~~~~~~~~~~~~
    In module 'UIKit' imported from /Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/Target Support Files/google_maps_flutter/google_maps_flutter-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIGraphicsImageRenderer.h:43:50: note: 'UIGraphicsImageRenderer' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 8.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(10.0)) @interface UIGraphicsImageRenderer : UIGraphicsRenderer
                                                     ^
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:178:7: note: enclose 'UIGraphicsImageRenderer' in an @available check to silence this warning
          UIGraphicsImageRenderer* renderer =
          ^~~~~~~~~~~~~~~~~~~~~~~
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:179:13: warning: 'UIGraphicsImageRenderer' is only available on iOS 10.0 or newer [-Wunguarded-availability]
              [[UIGraphicsImageRenderer alloc] initWithSize:_mapView.frame.size format:format];
                ^~~~~~~~~~~~~~~~~~~~~~~
    In module 'UIKit' imported from /Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/Target Support Files/google_maps_flutter/google_maps_flutter-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIGraphicsImageRenderer.h:43:50: note: 'UIGraphicsImageRenderer' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 8.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(10.0)) @interface UIGraphicsImageRenderer : UIGraphicsRenderer
                                                     ^
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:179:13: note: enclose 'UIGraphicsImageRenderer' in an @available check to silence this warning
              [[UIGraphicsImageRenderer alloc] initWithSize:_mapView.frame.size format:format];
                ^~~~~~~~~~~~~~~~~~~~~~~
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:181:53: warning: 'UIGraphicsImageRendererContext' is only available on iOS 10.0 or newer [-Wunguarded-availability]
          UIImage* image = [renderer imageWithActions:^(UIGraphicsImageRendererContext* context) {
                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'UIKit' imported from /Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/Target Support Files/google_maps_flutter/google_maps_flutter-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIGraphicsImageRenderer.h:39:50: note: 'UIGraphicsImageRendererContext' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 8.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(10.0)) @interface UIGraphicsImageRendererContext : UIGraphicsRendererContext
                                                     ^
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_flutter-0.5.25+3/ios/Classes/GoogleMapController.m:181:53: note: enclose 'UIGraphicsImageRendererContext' in an @available check to silence this warning
          UIImage* image = [renderer imageWithActions:^(UIGraphicsImageRendererContext* context) {
                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    6 warnings generated.
    /Users/sthomas/flutter/.pub-cache/hosted/pub.dartlang.org/cloud_functions-0.4.2+3/ios/Classes/CloudFunctionsPlugin.m:59:59: warning: comparison of distinct pointer types ('NSNumber *' and 'NSNull * _Nonnull') [-Wcompare-distinct-pointer-types]
        if (timeoutMicroseconds != nil && timeoutMicroseconds != [NSNull null]) {
                                          ~~~~~~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~
    1 warning generated.
    1 warning generated.
    /Users/sthomas/Desktop/Flutter/nothing_new/ios/Runner/AppDelegate.swift:12:5: error: use of unresolved identifier 'GeneratedPluginRegistrant'
        GeneratedPluginRegistrant.register(with: self)
        ^~~~~~~~~~~~~~~~~~~~~~~~~
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
    warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
```
The results of *flutter doctor --verbose*:
```
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.3 19D76, locale en-NZ)
    • Flutter version 1.12.13+hotfix.9 at /Users/sthomas/flutter
    • Framework revision f139b11009 (9 days ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/sthomas/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4, Build version 11E146
    • CocoaPods version 1.9.1

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] Connected device (1 available)
    • iPhone 11 Pro Max • F01FDB37-BA30-4A31-8F90-EE6DC28CD28F • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-4 (simulator)

• No issues found!
```

**Comment by nixon-nyx** [comment]
i got this issue too, any fixed?

**Comment by liyuqian** [comment]
The fix should be in https://github.com/flutter/flutter/pull/32266 . What's your Flutter version? Can you confirm if it has commit https://github.com/flutter/flutter/commit/969c6ef1dc1bf6351b538b9764fa6efa10744740 ?

**Comment by miguelcmedeiros** [comment]
@liyuqian 
I am having this issue in 1.17.0-dev.3.1.

**Comment by liyuqian** [comment]
1.17.0-dev.3.1 should include the fix. @xster @dnfield : maybe we regressed recently?

**Comment by andrewescutia** [comment]
any update or known workaround?
